pkg/kev/testdata/reconcile-env-var-override/docker-compose.kev.dev.yaml

```diff
@@ -1,6 +1,17 @@
 version: "3.9"
 services:
   db:
+    x-k8s:
+      workload:
+        replicas: 1
+        livenessProbe: 
+          type: exec
+          exec:
+            command: ["CMD", "echo", "Define healthcheck command for service db"]
```

Comment:
Ideally we should move toward probe command in a format accepted by the K8s and not use docker compose healthcheck test format, i.e. 

```yaml
exec:
  command:
    - echo
    - Define healthcheck...
```

When we parse docker compose healthcheck command it must be defined with one of `NONE`, `CMD`, `CMD-SHELL` - as per https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck. When we transform that healthcheck into something K8s understands we drop that first element, see: https://github.com/appvia/kev/blob/master/pkg/kev/converter/kubernetes/project_service.go#L675

Reply:
updated on all files